python/nav/web/templates/ipdevinfo/sensor-details.html

```diff
@@ -17,38 +17,7 @@
 {% endblock %}
 
 {% block footer_scripts %}
-<script>
- require([
-     "plugins/graphfetcher_controller",
-     "plugins/gauge",
- ], function(graphfetcher_controller, Gauge) {
-     $(function() {
-         var $gauge = $('.sensor-gauge');
-         var thresholds = $gauge.data('thresholds') ? $gauge.data('thresholds').split(',') : [];
-         var _gauge = new Gauge({
-             node: $gauge[0],
-             min: {{sensor.get_display_range.0}},
-             max: {{sensor.get_display_range.1}},
-             url: "{{sensor.get_graph_url}}&format=json",
-             symbol: ' ',
-             unit: "{{sensor.unit_of_measurement}}",
-             thresholds: thresholds,
-         });
-
-         $('#add-to-dashboard-button').on('click', function (event) {
-             event.preventDefault();
-             var $button = $(this);
-             var request = $.post($button.data('dashboardUrl'));
-             request.done(function() {
-                 $button.removeClass('secondary').addClass('success');
-             });
-             request.fail(function() {
-                 $button.removeClass('secondary').addClass('failure');
-             });
-         });
-     });
- });
-</script>
+<script src="{{ STATIC_URL }}js/src/sensor_details.js"></script>
```

Comment:
Everything looks fine-and-dandy to me, except this single bit. Even though my Javascript-fu is rusty, I know this is not the proper way to load JS modules in NAV: You're supposed to use require.js :)